source/agora/node/Ledger.d

```diff
@@ -600,6 +600,7 @@ public class Ledger
         NotEnoughValidators = "Enrollment: Insufficient number of active validators",
         MayBeValid = "May be valid",
         OnlyCoinbaseTX = "Transaction set only includes a Coinbase transaction",
+        TooManyMPVs = "More MPVs than active enrollments",
```

Comment:
What's MPV ?

Reply:
It's an acronym for `Missing Pre-image validator` but we need a properly shortened word for that, like `missing validator`.